doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -48,18 +154,159 @@ to keep your client decoupled from implementation details.
             "registrations_url": "/v2/registrations"
         }
 
-## Context Entities [/v2/entities{?type,pattern,limit,offset}]
-
-### List entities [GET /v2/entities{?limit,offset,count,type,id}]
-
-Detailed description
-
-+ Parameters
-    + limit: 20 (optional, number) - Limit the number of entities to be retrieved
-    + offset: 20 (optional, number) - Skip a number of records. Pagination
-    + type: Room (optional, string) - Only retrieve entities of this type
-    + id: Bcn-Welt (optional, string) - Retrieve entities by id
-    + idPattern: (optional, string) - Retrieve entities by pattern id
+## Context Entities [/v2/entities]
+
+### List entities [GET /v2/entities{?limit,offset,count,type,id,idPattern,q,geometry,coords,attrs}]
+
+Retrieves a list of entities which match criteria defined by the following parameters: `id`, `type`, `idPattern`, `q`,
+`geometry` and `coords` attribute (see below for a detailed description of these parameters). A given entity
+have to match all the criteria to be retrieved (i.e. criteria are combined in a logical AND way).
+
+<table style="width: 100%; margin: 12px 0 0 0;">
```

Comment:
I was wondering if there could be a way to avoid duplicating styles all over the document and using classes instead